.github/workflows/ci.yml

```
@@ -132,7 +132,10 @@ jobs:
       - name: Flake check
         run: nix flake check
       - name: Build all packages
-        run: nix build --accept-flake-config
+        run: |
+          # Pre-build monad-schedule without parallelism to avoid timing issues under load
+          nix build --accept-flake-config --max-jobs 1 .#monad-schedule-lib
+          nix build --accept-flake-config
```

Preferably these were 2 jobs
```suggestion
        run: |
          # Pre-build monad-schedule without parallelism to avoid timing issues under load
          nix build --accept-flake-config --max-jobs 1 .#monad-schedule-lib
          nix build --accept-flake-config
```